<launch>
  <node name="pwc_net" pkg="pwc_net" type="pwc_net_node">
    <param name="image_width" value="1248"/>
    <param name="image_height" value="376"/>
  </node>

  <node name="service_client" pkg="optical_flow_srvs" type="test_calculate_dense_optical_flow" required="true">
    <remap from="calc_flow" to="pwc_net/calculate_dense_optical_flow"/>
  </node>
</launch>
